apps/emqx/src/emqx_listeners.erl

```diff
@@ -355,14 +355,14 @@ do_stop_listener(Type, Id, #{bind := ListenOn}) when ?ESOCKD_LISTENER(Type) ->
 do_stop_listener(Type, Id, #{bind := ListenOn}) when ?COWBOY_LISTENER(Type) ->
     case cowboy:stop_listener(Id) of
         ok ->
-            wait_listener_stopped(ListenOn);
+            wait_listener_stopped(ListenOn, 0);
         Error ->
             Error
     end;
 do_stop_listener(quic, Id, _Conf) ->
     quicer:terminate_listener(Id).
 
-wait_listener_stopped(ListenOn) ->
+wait_listener_stopped(ListenOn, RetryMs) ->
```

Comment:
line 372 wont work if listener is not bound to loopback or all interfaces (0.0.0.0) ?

Reply:
Hm, wouldn't there will be `ip` in `Endpoint` in that case?

Reply:
true. my bad